src/rctmon/device_manager.py

```diff
@@ -362,7 +362,7 @@ def _cb_android_description(self, oid: int, value: str) -> None:
                           'g_sync.u_ptp_rms[1]', 'g_sync.u_ptp_rms[2]', 'grid_pll[0].f'], interval=10,
                          inventory=False, handler=self._cb_grid)
 
-            self.add_ids(['db.temp1', 'db.temp2', 'db.core_temp'], interval=60, inventory=False)
+            self.add_ids(['db.temp1', 'db.temp2', 'db.core_temp', 'battery.temperature'], interval=60, inventory=False, handler=self._cb_sensors)
```

Comment:
Querying for the battery temperature is already wired up in [battery_manager.py#L205](https://github.com/svalouch/rctmon/blob/main/src/rctmon/battery_manager.py#L205), so this would collide with it, and storing the value is in [battery_manager.py#L277](https://github.com/svalouch/rctmon/blob/main/src/rctmon/battery_manager.py#L277). It's exposed as `rctmon_battery_temperature` already.